app.rb

```diff
@@ -107,15 +107,25 @@ class InvalidChannelName < StandardError; end
     end
   end
 
-  get /(?<channel>\/[\w]+)?\/(?<project>[\w-]+)\/versions/ do
+  get /(?<channel>\/[\w]+)?\/(?<project>[\w-]+)\/full_list/ do
```

Comment:
IMO we should just skip this endpoint and redirect the legacy `/full_list`, `/full_client_list`, and `/full_server_list` to the new `/<CHANNEL>/<PROJECT>/versions`. These endpoints were 'internal' and never part of the public API: https://docs.chef.io/api_omnitruck.html